infra/modules/providers/azure/service-plan/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "random_id" "remotestate_account_name" {
+  byte_length = 6
+  keepers {
+    sa_account_ref = 1
+  }
+}
+
+resource "azurerm_resource_group" "svcplan" {
+  name     = "${var.resource_group_name == "" ? "${local.name}-cobalt-rg" : "${var.resource_group_name}"}"
+  location = "${var.resource_group_location}"
+  tags     = "${merge(map("Name", "${local.name}"), var.resource_tags)}"
+}
+
+resource "azurerm_app_service_plan" "svcplan" {
```

Comment:
This module should also support other settings like `tags`, `capacity`, `reserved`. Keep in mind that `reserved` needs to be set to true for linux based app service plans. Here's a reference for all settings supported for the TF resource https://www.terraform.io/docs/providers/azurerm/r/app_service_plan.html#argument-reference